library/src/scala/tasty/reflect/Printers.scala

```diff
@@ -92,6 +98,38 @@ trait Printers
     def showSymbol(symbol: Symbol)(implicit ctx: Context): String =
       new Buffer().visitSymbol(symbol).result()
 
+    def showFlags(flags: Flags)(implicit ctx: Context): String = {
+      val flagList = List.newBuilder[String]
+      if (flags.is(Flags.Protected)) flagList += "Flags.Protected"
+      if (flags.is(Flags.Abstract)) flagList += "Flags.Abstract"
+      if (flags.is(Flags.Final)) flagList += "Flags.Final"
+      if (flags.is(Flags.Sealed)) flagList += "Flags.Sealed"
+      if (flags.is(Flags.Case)) flagList += "Flags.Case"
+      if (flags.is(Flags.Implicit)) flagList += "Flags.Implicit"
+      if (flags.is(Flags.Erased)) flagList += "Flags.Erased"
+      if (flags.is(Flags.Lazy)) flagList += "Flags.Lazy"
+      if (flags.is(Flags.Override)) flagList += "Flags.Override"
+      if (flags.is(Flags.Inline)) flagList += "Flags.Inline"
+      if (flags.is(Flags.Macro)) flagList += "Flags.Macro"
+      if (flags.is(Flags.Static)) flagList += "Flags.javaStatic"
+      if (flags.is(Flags.Object)) flagList += "Flags.Object"
+      if (flags.is(Flags.Trait)) flagList += "Flags.Trait"
+      if (flags.is(Flags.Local)) flagList += "Flags.Local"
+      if (flags.is(Flags.Synthetic)) flagList += "Flags.Synthetic"
+      if (flags.is(Flags.Artifact)) flagList += "Flags.Artifact"
+      if (flags.is(Flags.Mutable)) flagList += "Flags.Mutable"
+      if (flags.is(Flags.FieldAccessor)) flagList += "Flags.FieldAccessor"
+      if (flags.is(Flags.CaseAcessor)) flagList += "Flags.CaseAcessor"
+      if (flags.is(Flags.Covariant)) flagList += "Flags.Covariant"
+      if (flags.is(Flags.Contravariant)) flagList += "Flags.Contravariant"
+      if (flags.is(Flags.Scala2X)) flagList += "Flags.Scala2X"
+      if (flags.is(Flags.DefaultParameterized)) flagList += "Flags.DefaultParameterized"
+      if (flags.is(Flags.Stable)) flagList += "Flags.Stable"
+      if (flags.is(Flags.Param)) flagList += "Flags.Param"
+      if (flags.is(Flags.ParamAccessor)) flagList += "Flags.ParamAccessor"
+      flagList.result().mkString(" | ")
```

Comment:
Private is not handled? In other places as well.

Reply:
Actually, the private flag is missing altogether 🤣. I will add it.

Reply:
Added